internal/integration/emitter.go

```diff
@@ -12,6 +12,8 @@ import (
 const (
 	defaultDeltaExpirationAge           = 5 * time.Minute
 	defaultDeltaExpirationCheckInterval = 5 * time.Minute
+	instrumentationName                 = "nri-prometheus"
+	instrumentationVersion              = "2.0.1"
```

Comment:
Should we link this variable to a config at build time?

`-v -ldflags '-X main.buildVersion=$(VERSION)'`

Otherwise it will be often outdated

Reply:
Oops, i think we probably don't need that. I just saw right now where we are setting the version to the integration, so we don't need these 2 consts